### PR TITLE
Add quantized::convtranspose2d op NNAPI converter

### PR DESCRIPTION
**Pull request by axitkhurana**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #63914
* #63913

Summary:

As per title

Test Plan:

pytest test/test_nnapi.py::TestNNAPI::test_conv2d_transpose

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D30531889](https://our.internmc.facebook.com/intern/diff/D30531889)

### CODE REVIEW

**Comment by axitkhurana** [comment]
@axitkhurana has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30531889).

**Comment by axitkhurana** [comment]
@axitkhurana has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30531889).

File: test/test_nnapi.py [hunk 1]
```diff
@@ -540,34 +540,29 @@ def test_conv2d(self):
...
-                    model.eval()
+                    model = torch.nn.quantized.ConvTranspose2d(in_ch, out_ch, kernel)
                     model.qconfig = torch.quantization.get_default_qconfig('qnnpack')
-                    model = torch.quantization.prepare(model)
-                    model(inp)
-                    model = torch.quantization.convert(model)
                     inp = qpt(inp, 1.0 / 16, 128)
                     # I've seen numerical differences between QNNPACK and NNAPI,
-                    # but never more than 1 quantum, and never more than ~1% of
+                    # but never more than 1 quantum, and never more than ~10% of
```
Comment (dreiss) [inline test/test_nnapi.py:562]: Wait, is this for real?  9% deviation sounds like a bug

Comment (axitkhurana) [reply test/test_nnapi.py:562]: Yeah did see up to 10% of elements being different up to 1 quantum, I can share a few examples

Comment (dreiss) [reply test/test_nnapi.py:562]: Oh, wait.  I was reading this as 10% delta.  10% of 1 quantum, I guess that's life.

**Review by dreiss — APPROVED** [review]

**Pull request closed without merging**